app/Ptr/PtrControllerTest.php

```diff
@@ -156,6 +158,16 @@ public function testListUpdatesOnCreate() {
     });
   }
 
+  public function testPtrNotAssociatedWithIpOnCreate(){
+    $this->asAdminWithPermissions(static::PERMISSIONS, function () {
+      $this->mockDns('test_create', '1.1.1.3', 1);
+      $ip = "9.9.9.9";
+      $this->tryCreate(['ip' => $ip]);
+      $this->assertResponseStatus(409);
+      $this->assertMessageContains('Invalid IP, IP is not associated with Ptr');
```

Comment:
This error message is not as clear as it could be, it would be helpful to tell the user specifically what they can do to fix the problem. I think this would be `"Please ensure that <PTR hostname> has an A or AAAA DNS record to <PTR IP>"`.

Reply:
Looks like this belongs on the other PR, I'll take this there.